publiq-ui/pub-page-title.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <pub-title size="1" class="page-title"><slot /></pub-title>
+  <pub-title :size="1" class="page-title"><slot /></pub-title>
```

Comment:
Curious: Does it need the `:` because it expects a number instead of a string? Or is there another reason?

Reply:
Ah I see we also had to change it in another PR to make it pass a number. Makes sense 👍